src/lib/KeyStoreIndexeddb.js

```diff
@@ -215,15 +215,15 @@ class KeyStore {
 
         const keys = await accountStore.dangerousListPlain();
 
-        for (const key of keys) {
+        keys.forEach(async key => {
```

Comment:
I think it's also ok to use for ... of...

Reply:
Explained here: https://github.com/airbnb/javascript/issues/1271